src/conmon.c

```diff
@@ -547,6 +547,26 @@ static bool has_v1_oom()
 	return false;
 }
 
+/* write the appropriate files to tell the caller there was an oom event
+ * this can be used for v1 and v2 OOMS */
+static int write_oom_files() {
+	_cleanup_close_ int oom_fd = -1;
```

Comment:
Can we check if this works with us returning oom_fd here?

Reply:
ope, it did not. Fixed